src/debugging.js

```diff
@@ -77,9 +77,11 @@ export function getConfig(debugging) {
 }
 config.getConfig('debugging', ({debugging}) => getConfig(debugging));
 
-export function sessionLoader(storage = window.sessionStorage) {
+export function sessionLoader() {
   let overrides;
   try {
+    if (!window.localStorage || !window.sessionStorage) throw "exception";
+    storage = window.sessionStorage || window.localStorage;
```

Comment:
if one of the problems is that just accessing `sessionStorage` might throw an exception, while `localStorage` is still available, this is still a problem because the second part of the `||` will never be reached.  You'd have to do something like:
`let storage;`
`try {  storage = window.sessionStorage || window.localStorage }`
`catch(e) { storage = window.localStorage }`